perceval/backends/github.py

```diff
@@ -254,6 +276,25 @@ def __get_headers(self):
             headers = {'Authorization': 'token ' + self.token}
             return headers
 
+    def __get_github(self, url, params=None, headers=None):
+        """ GET HTTP caring of rate limit """
+        if self.rate_limit is not None and self.rate_limit <= RATE_LIMIT_MIN:
+            reset_seconds = (self.rate_limit_reset-datetime.utcnow()).seconds+1
+            cause = "GitHub rate limit exhausted."
+            if self.sleep_for_rate:
+                logging.info("%s Waiting %i secs for rate limit reset." % (cause, reset_seconds))
+                sleep(reset_seconds)
+            else:
+                ex = RateLimitError(cause=cause, reset_seconds=reset_seconds)
```

Comment:
Yes, I have the "ex" var for testing the attribute ... but it is not needed anymore.